src/main/java/io/pravega/perf/performance.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package com.emc.pravega.perf;
+package io.pravega.perf;
```

Comment:
Please, rename this class as `Performance`, as classes and interfaces in Java start with capitals.